### PR TITLE
tiup: add instructions for the --local option

### PR DESCRIPTION
**Pull request by huanglao2002**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

为方便入门者使用tiup工具创建tidb集群，新增--local选项说明，使用该参数可以创建一个简单本地集群。
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
Hi @huanglao2002, may I know if there is a code change PR related to this doc change?

**Review by kaaaaaaang — APPROVED** [review]

File: tiup/tiup-component-cluster-template.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ tiup cluster template [flags]
 - 输出详细的拓扑模版，该模版会以注释的形式带上可配置的参数。在命令中添加该选项，可开启该选项。
 - 如果不指定该选项，默认输出最简单的拓扑模版。
 
+### --local
+
+- 输出简单本地集群的拓扑模版，该模版可以直接使用，global参数也可以按需调整。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-template.md:33]:
```suggestion
- 输出本地集群的简单拓扑模版，该模版可以直接使用，global 参数也可以按需调整。
```

File: tiup/tiup-component-cluster-template.md [hunk 2]
```diff
@@ -28,6 +28,11 @@ tiup cluster template [flags]
...
+- 该模板会创建一个pd服务、一个tidb服务、一个tikv服务、一个monitoring服务、一个grafana服务。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-template.md:34]:
```suggestion
- 该模板会创建一个 PD 服务、一个 TiDB 服务、一个 TiKV 服务、一个 monitoring 服务、一个 Grafana 服务。
```

**Comment by huanglao2002** [comment]
> Hi @huanglao2002, may I know if there is a code change PR related to this doc change?

不涉及

File: tiup/tiup-component-cluster-template.md [hunk 3]
```diff
@@ -28,6 +28,11 @@ tiup cluster template [flags]
 - 输出详细的拓扑模版，该模版会以注释的形式带上可配置的参数。在命令中添加该选项，可开启该选项。
 - 如果不指定该选项，默认输出最简单的拓扑模版。
 
+### --local
+
+- 输出本地集群的简单拓扑模版，该模版可以直接使用，global 参数也可以按需调整。
```
Comment (qiancai) [inline tiup/tiup-component-cluster-template.md:33]:
```suggestion
- 输出本地集群的简单拓扑模版，该模版可以直接使用，该模版中的 global 参数也可以按需调整。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**